textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/utils/Utils.java

```diff
@@ -0,0 +1,65 @@
+package edu.uci.ics.textdb.common.utils;
+
+import java.text.ParseException;
+import java.util.Date;
+
+import org.apache.lucene.document.DateTools;
+import org.apache.lucene.document.DateTools.Resolution;
+import org.apache.lucene.document.Field.Store;
+import org.apache.lucene.document.IntField;
+import org.apache.lucene.index.IndexableField;
+
+import edu.uci.ics.textdb.api.common.IField;
+import edu.uci.ics.textdb.common.field.DateField;
+import edu.uci.ics.textdb.common.field.DoubleField;
+import edu.uci.ics.textdb.common.field.FieldType;
+import edu.uci.ics.textdb.common.field.IntegerField;
+import edu.uci.ics.textdb.common.field.StringField;
+
+public class Utils {
+    public static IField getField(FieldType fieldType, String fieldValue) throws ParseException{
+        IField field = null;
+        switch (fieldType) {
+            case STRING:
+                field = new StringField(fieldValue);
+                break;
+            case INTEGER:
+                field = new IntegerField(Integer.parseInt(fieldValue));
+                break;
+            case DOUBLE:
+                field = new DoubleField(Double.parseDouble(fieldValue));
+                break;
+            case DATE:
+                field = new DateField(DateTools.stringToDate(fieldValue));
+                break;
+            default:
+                break;
+        }
+        return field;
+    }
+
+    public static IndexableField getLuceneField(FieldType fieldType,

```

Comment:
Does the name need to specify the dependency on Lucene?  That is, is it better to change "getLuceneField" to "getField"?